docs/index.rst

```diff
@@ -11,6 +11,7 @@ DebOps Policies and Guidelines
    distributed-development
    code-signing-policy
    security-policy
+   planed-policy-changes
```

Comment:
Shouldn't this be 'planned-policy-changes'? https://oilpatchwriting.wordpress.com/2010/08/12/planed-vs-planned/